tx/textual/valuerenderer/valuerenderer.go

```diff
@@ -45,8 +46,8 @@ func NewTextual(q CoinMetadataQueryFn) Textual {
 	return t
 }
 
-// GetValueRenderer returns the value renderer for the given FieldDescriptor.
-func (r Textual) GetValueRenderer(fd protoreflect.FieldDescriptor) (ValueRenderer, error) {
+// GetFieldValueRenderer returns the value renderer for the given FieldDescriptor.
+func (r Textual) GetFieldValueRenderer(fd protoreflect.FieldDescriptor) (ValueRenderer, error) {
```

Comment:
In this function's `case fd.Kind() == protoreflect.MessageKind:` can we simply call GetMessageValueRenderer?

Reply:
No - we still want to handle repeated fields here - logic that we don't want in `GetMessageValueRenderer()`. After the repeated renderer lands we can look at refactoring.